core/src/main/resources/error/error-classes.json

```diff
@@ -125,7 +120,7 @@
   },
   "CANNOT_PARSE_PROTOBUF_DESCRIPTOR" : {
     "message" : [
-      "Error parsing file <descFilePath> descriptor byte[] into Descriptor object."
+      "Error parsing descriptor bytes into Protobuf FileDescriptorSet."
```

Comment:
We don't have the file name for this error. 
Failure to read from the file results in `QueryCompilationErrors.cannotFindDescriptorFileError()`